sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -71,8 +71,11 @@ public void StopLoggingEvents()
         /// This will run prior to the start of each test.
         /// </summary>
         [SetUp]
-        public void SetupEventsForTest() =>
+        public void SetupEventsForTest()
```

Comment:
Would this be better in `ClientTestBase` so it would apply to all tests, not just Storage?

Reply:
Yes, if I can find a way to improve the output.